AritDeveloperCircle/findyourpeople
# Fix Broken Prod: Individual Listing not displaying

## What type of PR is this? (check all applicable)

  
  
## Description of what this Pull Request does
After the latest merges, `main` branch is broken; namly, the individual listing does not display.
.
.
<img width="1680" alt="Screen Shot 2023-06-20 at 8 21 32 PM" src="https://github.com/AritDeveloperCircle/findyourpeople/assets/32520970/a683c2d4-ac7e-40e0-83bc-410a74903f7c">

  
## QA Instructions, Screenshots, Recordings
To QA the fix, please checkout this branch and ensure that when you click on a `View More` link from the Landing Page, the individual listing displays correctly, and no other features are broken.
